src/epr/client.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: © 2024 Brett Smith <xbcsmith@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import logging
+from typing import Any, Optional
+from urllib.parse import urljoin
+
+import urllib3
+
+from .common import EnhancedJSONEncoder
+
+urllib3.disable_warnings()
+
+logger = logging.getLogger(__name__)
+
+
+class Client(object):
```

Comment:
Suggest keeping `search_*` & `get_*` methods as-is and making remainder "private" (`__` prefix?) so consumers know what's intended for use.

Reply:
_ is gentleman's private. To be clear epr-python is intended to be a library + client. The CLI as a convenience/example. While not obvious now there may be uses for the functions later.